helix-core/src/comment.rs

```diff
@@ -317,56 +341,95 @@ pub fn split_lines_of_selection(text: RopeSlice, selection: &Selection) -> Selec
 mod test {
     use super::*;
 
-    #[test]
```

Comment:
This diff is pretty hard to follow. I think these can be reset together with the function I mentioned above?

Reply:
> This diff is pretty hard to follow.

What I'm doing here is just splitting up this one test function, which tests multiple things, into smaller tests which only test one single thing.
I see the following advantages here:
- more isolated tests
- more test functions can run in parallel => faster testing

Do you still want to merge them back into one single test function?

Reply:
Ah I thought you made the change due to the change you made to the function above.

Taking another look it seems like a good change. I don't think the parallelism matters for these tiny tests but having multiple (named) independence tests (that can pass/fail independently) is easier to work with